### PR TITLE
9493 display consolidated case docket numbers, save and serve

### PR DESCRIPTION
**Pull request by rosiehsmith**
For save and serve immediately flow, update docket numbers on coversheet to display for all selected cases in the consolidated group
image

### CODE REVIEW

File: shared/src/business/entities/EntityConstants.js [hunk 1]
```diff
@@ -434,7 +434,13 @@ const TRACKED_DOCUMENT_TYPES = {
 const SINGLE_DOCKET_RECORD_ONLY_EVENT_CODES = flatten([
   ...Object.values(DOCUMENT_EXTERNAL_CATEGORIES_MAP),
 ])
-  .filter(internalEvent => internalEvent.caseDecision)
+  .filter(externalEvent => externalEvent.caseDecision)
```
Comment (jamesobrooks) [inline shared/src/business/entities/EntityConstants.js:437]: :question: Was `internalEvent` a bad name here to begin with?

File: shared/src/business/useCaseHelper/consolidatedCases/formatConsolidatedCaseCoversheetData.test.js [hunk 2]
```diff
@@ -50,6 +50,20 @@ describe('formatConsolidatedCaseCoversheetData', () => {
...
 
+  it('should not add any consolidated group information to the coverhseet when the document has only been filed on the lead case', async () => {
+    const result = await formatConsolidatedCaseCoversheetData({
+      applicationContext,
+      caseEntity: MOCK_CASE,
+      coverSheetData: {},
+      docketEntryEntity: {
+        ...mockDocketEntry,
+        docketEntryId: 'b0efe7fd-a8d6-4eb8-bf66-857bcb700483',
+      },
```
Comment (jamesobrooks) [inline shared/src/business/useCaseHelper/consolidatedCases/formatConsolidatedCaseCoversheetData.test.js]: 🤔  Maybe I'm just lacking context, but looking at this I don't really see how this means we've only filed on the lead case. Is it the docketEntryId specifically? If so, could we use a well-named constant to indicate that?

**Review by jamesobrooks — COMMENTED** [review]
Looks good, but I had a couple of questions.

File: shared/src/business/entities/EntityConstants.js (hunk 1 above)
Comment (rosiehsmith) [reply shared/src/business/entities/EntityConstants.js:437]: yes, though internal filing events contains the event codes we wanted to exclude, it's more accurate for paper filings to use external filing events and we're grabbing from the `DOCUMENT_EXTERNAL_CATEGORIES_MAP`

File: shared/src/business/useCaseHelper/consolidatedCases/formatConsolidatedCaseCoversheetData.test.js (hunk 2 above)
Comment (rosiehsmith) [reply shared/src/business/useCaseHelper/consolidatedCases/formatConsolidatedCaseCoversheetData.test.js]: fair point. trying to name that constant is hard but we're making an attempt! it needs to be a docket entry that isn't multi-docketed, meaning it's not returned in any consolidatedCases from `getCasesByLeadDocketNumber`.

**Pull request merged**